src/DataStructures/Tensor/Python/tnsr.py

```diff
@@ -34,6 +34,7 @@
 ijj = TensorMeta("ijj")
 iJJ = TensorMeta("iJJ")
 Ijj = TensorMeta("Ijj")
+ijaa = TensorMeta("ijaa")
```

Comment:
There a section with mixed-indices tensors a few lines below, move it there I think